src/SFA.DAS.ApplyService.sln

```diff
@@ -49,7 +49,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.ApplyService.Types"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.ApplyService.EmailService.UnitTests", "SFA.DAS.ApplyService.EmailService.UnitTests\SFA.DAS.ApplyService.EmailService.UnitTests.csproj", "{134D8E1C-F001-471C-B07B-C8120392D4C1}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.ApplyService.Data.UnitTests", "SFA.DAS.ApplyService.Data.UnitTests\SFA.DAS.ApplyService.Data.UnitTests.csproj", "{C7D71EBB-BFCB-41D9-BE7F-F3E1225EE876}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.ApplyService.Data.UnitTests", "SFA.DAS.ApplyService.Data.UnitTests\SFA.DAS.ApplyService.Data.UnitTests.csproj", "{C7D71EBB-BFCB-41D9-BE7F-F3E1225EE876}"
```

Comment:
Any reason as to why this needs changing to be a Web App (9A19103F-16F7-4668-BE54-9A1E7A4F7556) ?

Reply:
I'll reset this